drivers/pinmux/stm32/pinmux_stm32h7.h

```diff
@@ -507,7 +508,7 @@
 	(STM32_PINMUX_ALT_FUNC_8 | STM32_PUSHPULL_PULLUP)
```

Comment:
why then leaving USART6_RTS and CTS wrongly on  AF8,  AF7 is the correct setting.

Idem for PG15 below

Reply:
You are absolutely right ! I didn't have a look at the other pins as I didn't test them. I have submitted a new commit with all of the USART6 AF corrections. :+1 Thanks for your feedback